docs/tutorials/build_a_node_in_rust.md

```diff
@@ -696,35 +696,35 @@ fn handle_ldk_event(..) {
 ```rust
 let params = ProbabilisticScoringParameters::default();
 
-let scorer = ProbabilisticScorer::new(params, Arc:clone(&network_graph));
+let scorer = Arc::new(Mutex::new(ProbabilisticScorer::new(params, Arc::clone(&network_graph))));
 ```
 
 **Dependencies** `NetworkGraph`
 
 **Reference** [`ProbabilisticScorer` docs](https://docs.rs/lightning/*/lightning/routing/scoring/type.ProbabilisticScorer.html),
-[LDK sample node ProbabilisticScorer example](https://github.com/lightningdevkit/ldk-sample/blob/main/src/main.rs#L612)
+[LDK sample node ProbabilisticScorer example](https://github.com/lightningdevkit/ldk-sample/commit/6f6473bf1bb4a1b2149a9a5c3616c39653dec144#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR592)
```

Comment:
Similar here regarding link consistency. The sample code is now under disk.rs.

That said, not sure if it's worth referencing it here given how simple the code is. The sample does show how to read it from disk, though. Will leave it up to you whether the links are needed.

Reply:
Went with link to the sample code here.